js/adevar/AdevarTile.js

```diff
@@ -16,11 +16,19 @@ var AdevarTileCode = {
 	vanguard: "Vanguard",
 	reflection: "WatersReflection",
 	iris: "Iris",
-	irisSF: "IrisSecondFace",
+	irisSF: "Iris Second Face",
```

Comment:
These are the "tile codes" which are used in the notation and also the image file names, so I'll be removing the spaces in the name here. I'd like to shorten these up before "going live". Doing so will break testing games, however.